cmd/admission/app/configure/configure.go

```diff
@@ -79,78 +94,146 @@ func (c *Config) CheckPortOrDie() error {
 	return nil
 }
 
-// PatchMutateWebhookConfig patches a CA bundle into the specified webhook config.
-func PatchMutateWebhookConfig(client admissionregistrationv1beta1client.MutatingWebhookConfigurationInterface,
-	webhookConfigName, webhookName string, caBundle []byte) error {
-	config, err := client.Get(webhookConfigName, metav1.GetOptions{})
-	if err != nil {
-		return err
+func useGeneratedNameIfRequired(configured, generated string) string {
+	if configured != "" {
+		return configured
 	}
-	prev, err := json.Marshal(config)
+	return generated
+}
+
+// RegiesterWebhooks register webhooks for admission service
+func RegiesterWebhooks(c *Config, clienset *kubernetes.Clientset, cabundle []byte) error {
+	// Get the admission service
+	admissionService, err := clienset.CoreV1().Services(
```

Comment:
I would suggest removing this dependency, think if admission process running here, but no service is created for it.

Reply:
The webhook configuration needs work with its backed service. see [here](https://github.com/volcano-sh/volcano/blob/789f95002e6f6f9e887414765e8d8e1cf1c20b90/installer/helm/chart/volcano/templates/admission-config.yaml#L11), or maybe we need waiting sometime until its get exist(ready)?

Reply:
I know that it need the service. Doesnot matter, without service, the call will fail. Once service comes up, it can succeed.